src/domUpdates.js

```diff
@@ -72,6 +72,8 @@ let domUpdates = {
     </header>
     <span class='recipe-name ${recipe.id}'>${recipe.name}</span>
       <img tabindex='0' class='card-picture ${recipe.id}'src='${recipe.image}' alt='Food from recipe'>
+      <label>Buy Needed Ingredients</label>
+      <input type="checkbox" class="bought-ingridients ${recipe.id}">
     </div>`
```

Comment:
I like this added functionality of indicating ingredient purchase. Let's think about moving it to the "To Cook" view since the list of needed ingredients and cost is also there. This will also keep the home page uncluttered.

Reply:
Done! I now have two buttons on the "To Cook" sections instead of check boxes so that once cooked they can be disbaled.